developer/plugin-development.md

```diff
@@ -60,7 +60,7 @@ It is recommended to use the new v1 plugin API for writing new plugins.
 
 ### Send a patch or fork?
 
-If you have a problem with any existing plugins or new feature idea, sending a
+If you have a problem with any built-in plugins or a new feature idea, sending a
```

Comment:
`built-in` means `fluentd` gem bundled plugins like `in_forward`. This section is for any plugins including 3rd party plugin, so I think `existing` is better.